mdflynn/game-sleuth
# Feature/search by name

## Is this a fix or a feature?
This is a feature

## What is the change?
- Adds the ability to search by game name in the search form
- Adds testing for new input field
- Updates App integration testing for SearchForm updates

## What does it fix?
No fix this PR

## Where should the reviewer start?
Lines 44-60 in the SearchForm.tsx

## How should this be tested?
Run `npm start` and navigate to localhost: 3000 to see new functionality
Run `npm test` to see passing test suite

## Relevant Issues
#39
